json-logic/example.json

```diff
@@ -0,0 +1,318 @@
+{
+  "meta": {
+    "author": "Alexander",
+    "languages": [
+      "DE"
+    ],
+    "name": "Example",
+    "version": "0.1"
+  },
+  "questions": [
+    {
+      "id": "q1_contact",
+      "text": "Gab es Kontakt zu bestätigten Fällen?",
+      "type": "Boolean"
```

Comment:
just for clarity: We are using camleCase everywhere but not here. Should we use it? 

Info: FHIR is also using camelcase.

Reply:
Because these are constants, not attributes. 

We can also move them to camel case or all-capital letters, I don't really care.

Reply:
I think following FHIR here again and using camelCase would be the best choice.